packages/fxa-settings/src/components/images/index.tsx

```diff
@@ -39,7 +39,7 @@ type PreparedImageProps =
   | PreparedImageAriaVisibleProps;
 
 export const PreparedImage = (props: PreparedImageProps) => {
-  const { className = 'w-3/5 mx-auto', ariaHidden, Image } = props;
+  const { className, ariaHidden, Image } = props;
```

Comment:
Removed the classnames that were only used for storybook - they made it a bit harder to see the boundaries of the SVGs.